### PR TITLE
Remove unneeded config from Simple saga sample

### PR DESCRIPTION
**Pull request by andreasohlund**
Please review @SimonCropp

### CODE REVIEW

File: samples/saga/simple/Core_6/Sample/Program.cs [hunk 1]
```diff
@@ -16,23 +14,14 @@ static void Main()
     {
         Console.Title = "Samples.SimpleSaga";
         var endpointConfiguration = new EndpointConfiguration("Samples.SimpleSaga");
-        endpointConfiguration.UseSerialization<JsonSerializer>();
```
Comment (SimonCropp) [inline samples/saga/simple/Core_6/Sample/Program.cs]: this is in keeping with https://docs.particular.net/samples/#technology-choices-serializer

File: samples/saga/simple/Core_6/Sample/Program.cs [hunk 2]
```diff
@@ -16,23 +14,14 @@ static void Main()
...
-        endpointConfiguration.EnableInstallers();
-
-        var recoverability = endpointConfiguration.Recoverability();
-        recoverability.DisableLegacyRetriesSatellite();
```
Comment (SimonCropp) [inline samples/saga/simple/Core_6/Sample/Program.cs]: without this the learning transport creates a redundant `Retries` directory structure

File: samples/saga/simple/Core_6/Sample/Program.cs (hunk 1 above)
Comment (andreasohlund) [inline samples/saga/simple/Core_6/Sample/Program.cs]: Ok I see, will add it back

File: samples/saga/simple/Core_6/Sample/Program.cs (hunk 2 above)
Comment (andreasohlund) [inline samples/saga/simple/Core_6/Sample/Program.cs]: I'll pr the core to avoid this for the learning transport so that we don't need this crap here

**Pull request merged**